matchbox_socket/src/ggrs_socket.rs

```diff
@@ -49,3 +49,11 @@ impl ggrs::NonBlockingSocket<PeerId> for WebRtcSocket {
         messages
     }
 }
+
+impl WebRtcSocketBuilder {
+    /// Adds a new channel configured correctly for usage with GGRS to the [`WebRtcSocket`].
```

Comment:
Should perhaps mention the feature must be enabled?

Reply:
Possibly, went without to stay in line with `players`. Have seen libraries do both

Reply:
Okay, let's maybe do it separately, then